File: OCH_Tao/ABC/B/356.py
N,M = map(int,input().split())
A = list(map(int,input().split()))
for i in range(N):
  X = list(map(int,input().split()))
  for j in range(M):
    A[j]-=X[j]
print("Yes" if all([i<=0 for i in A]) else "No")

File: OCH_Tao/ABC/B/356.js
const input = require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n");
const M = parseInt(input.shift().split(" ")[1], 10);
let A = input.shift().split(" ").map(x => parseInt(x, 10));
for (const i of input) {
  const X = i.split(" ").map(x => parseInt(x, 10));
  for (let i = 0; i < M; i++) {
    A[i] -= X[i];
  }
}
console.log(A.every(x => x <= 0) ? "Yes" : "No");